tests/tests/swfs/avm2/matrix3d/Test.as

```diff
@@ -56,6 +56,12 @@
 			vOut = mat.transformVector(v);
 			trace(vOut.x, vOut.y, vOut.z, vOut.w);
 
+			var vecs:Vector.<Number> = Vector.<Number>([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]);
```

Comment:
Could you add calls with null arguments, within catch blocks that print the exception? Thanks!